bootstrap

```diff
@@ -156,6 +156,9 @@ while (true) {
   }
 
   if (strlen($body) > 0) {
+    if($event['httpMethod'] === 'POST'){
```

Comment:
I think this works if the content type is `application/x-www-form-urlencoded`. But I'm not sure it works if the body is instead `application/json`. The docs make it sound like the body will be incompatible with what `CURLOPT_POSTFIELDS` expects (see http://php.net/manual/en/function.curl-setopt.php).

Can you verify what happens if you post a JSON payload with `Content-Type: application/json` set? If this change breaks posting non-form-urlencoded bodies, then please add a condition here that checks the content type.

Reply:
OK, I verified some test case.

## Verification code
```
<?php
$input = file_get_contents("php://input");
echo $input;
```

## verify at local machie

### prepare builtin server
```
php -S localhost:8080 index.php
```

### send request
```
 curl -X POST http://localhost:8080 -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
{"key": "val"}
```

## verify at AWS using arn:aws:lambda:${AWS::Region}:887080169480:layer:php71:5

### send request
```
 curl -X POST https://xxxxx.execute-api.ap-northeast-1.amazonaws.com/Prod/index.php -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
```
empty response body returned 


## verify at AWS using my layer that patched the issue 

### send request
```
 curl -X POST https://xxxxx.execute-api.ap-northeast-1.amazonaws.com/Prod/index.php -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
{"key": "val"}
```


I think, It was not send Json Payload from before


I'm not good at English.

Reply:
Awesome! Looks great, thanks for confirming the body still comes through!